File: TeamCode/src/main/java/brickbot/old/quickstart/follower/Pose.java
package brickbot.old.quickstart.follower;

public class Pose {
    private double x;
    private double y;
    private double heading;

    public Pose(double x, double y, double heading){
        this.x = x;
        this.y = y;
        this.heading = heading;
    }

    public double getX() {
        return x;
    }

    public double getY() {
        return y;
    }

    public double getHeading() {
        return heading;
    }
}

File: TeamCode/src/main/kotlin/brickbot/quickstart/follower/Pose.kt
package brickbot.quickstart.follower

import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit
import org.firstinspires.ftc.robotcore.external.navigation.DistanceUnit
import org.firstinspires.ftc.robotcore.external.navigation.Pose2D

class Pose(
    var x: Double,
    var y: Double,
    var heading: Double
) {
    constructor(pose2D: Pose2D): this(pose2D.getX(DistanceUnit.CM), pose2D.getY(DistanceUnit.CM), pose2D.getHeading(AngleUnit.DEGREES))
}